### ISSUE TITLE
fakefs 2.5.0 testsuite fails with ruby 3.3.2 / irb 1.3.1 and above

### ISSUE THREAD

**Issue by mtasaka** [fakefs/fakefs#495]
Recently ruby 3.3.2 is released (and 3.3.3 is then released), which upgrades irb to 1.3.1, and now fakefs 2.5.0 testsuite fails like:

```
 1) Failure:
IrbTest#test_setup_irb [test/irb_test.rb:18]:
Expected true to be nil.
737 runs, 1478 assertions, 1 failures, 0 errors, 1 skips
```

This is because of `IRB.setup` change by this commit:

https://github.com/ruby/irb/commit/06b2d00dd3ceed2a38a0c2c8af890facf87ef649#diff-f9b03212edb2ad71662a5aa868477764516ea8e90bc2ed556f26e73f2511cf07R60

fakefs testsuite expects that `IRB.setup` returns `nil` :
https://github.com/fakefs/fakefs/blob/f38d42295a62301b73a5017e395715de71d6afa2/test/irb_test.rb#L18
but now `IRB.setup` returns `true` instead of `nil`, due to the above change.

**Comment by mtasaka** [comment]
ref: https://github.com/ruby/irb/pull/850

**Comment by grosser** [comment]
thx https://github.com/fakefs/fakefs/pull/496

**Comment by santiagorodriguez96** [comment]
Hey @grosser! Thank you for fixing this! 

Could you please consider cutting a new release that includes the necessary changes? The app that I'm working on requires this fix in order to upgrade to Ruby 3.3 🙂 

Thank you so much for you time and effort!

**Comment by grosser** [comment]
that's just a test change, releasing it won't do anything ...
